.obs/workflows.yml

```diff
@@ -3,3 +3,12 @@ workflow:
     - branch_package:
         source_project: OBS:Server:Unstable
         source_package: obs-server
+  filters:
+    architectures:
+      only:
+        - x86_64
+    repositories:
+      only:
+        - SLE_12_SP5
+        - SLE_15_SP1
+        - SLE_15_SP2
```

Comment:
@dmarcoux I know that these are the repositories we used to report the status for, but the latest stable release for SLE 15 is SP3, maybe we should include it to the list, what do you think?

Reply:
For now, I would keep it to what we had. SLE15 SP3 is currently unresolvable.

Reply:
okay, not resolvable at the moment